bundle/deploy/terraform/init.go

```diff
@@ -43,8 +43,11 @@ func (m *initialize) findExecPath(ctx context.Context, b *bundle.Bundle, tf *con
 		return tf.ExecPath, nil
 	}
 
+	// Resolve the version of the Terraform CLI to use.
+	tv := GetTerraformVersion(ctx)
```

Comment:
Why panic over standard error diagnostic? IMO stacktrace is not helpful there since it's not programmer's error.

Reply:
Makes sense. I'm adding error propagation to the relevant functions.